sessionctx/stmtctx/stmtctx.go

```diff
@@ -38,6 +39,13 @@ const (
 	WarnLevelNote = "Note"
 )
 
+var taskIDAlloc uint64
```

Comment:
taskIDAlloc is a server-level variable here.
Can we define it as a session-level variable to reduce the contention on it?